src/Interfaces/referral/Referral.ts

```diff
@@ -0,0 +1,8 @@
+import ReferredUser from './ReferredUser';
+
+export default interface Referral {
+    code: string;
+    totalEarned: number;
+    referredUsers: Array<ReferredUser> | [];
```

Comment:
Should this be 
```suggestion
    referredUsers: Array<ReferredUser>;
```

`[]` isn't really a type that we use

Reply:
I did that because referredUsers can be an empty array...